{"frames":[{"func":"INLA::inla(formula, family = \"binomial\", control.family = list(control.link = list(model = \"logit\")), \n    data = dat, Ntrials = m, control.predictor = list(compute = TRUE, \n        link = 1), verbose = TRUE)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"inla.inlaprogram.has.crashed()","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"stop(\"The inla-program exited with an error. Unless you interupted it yourself, please rerun with verbose=TRUE and check the output carefully.\\n  If this does not help, please contact the developers at <help@r-inla.org>.\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in inla.inlaprogram.has.crashed() : \n  The inla-program exited with an error. Unless you interupted it yourself, please rerun with verbose=TRUE and check the output carefully.\n  If this does not help, please contact the developers at <help@r-inla.org>.\n"}